### PR TITLE
Allowing users to restart time entries

### PR DESCRIPTION
**Pull request by juanultimate**
Allow users to restart a time entry.

### CODE REVIEW

File: src/app/modules/time-entries/pages/time-entries.component.ts [hunk 1]
```diff
@@ -42,33 +44,37 @@ export class TimeEntriesComponent implements OnInit {
...
     }
     this.store.dispatch(new entryActions.LoadEntries(new Date().getMonth() + 1));
   }
 
-  doSave(entry) {
-    entry.start_date = new Date(entry.start_date).toISOString();
-    if (entry.end_date !== null && entry.end_date !== undefined) {
-      entry.end_date = new Date(entry.end_date).toISOString();
+  doSave(event: SaveEntryEvent) {
+    console.log(event);
```
Comment (enriquezrene) [inline src/app/modules/time-entries/pages/time-entries.component.ts:65]: do we need this?

File: src/app/modules/time-entries/pages/time-entries.component.html [hunk 2]
```diff
@@ -64,13 +64,15 @@ <h5 class="modal-title" id="exampleModalLongTitle">{{ entryId ? 'Edit Entry' : '
...
           [entryToEdit]="entry"
           (saveEntry)="saveEntry($event)"
-        ></app-details-fields>
+        >  Active TE: {{activeTimeEntry | json}}
+        </app-details-fields>
       </div>
     </div>
   </div>
+
+  Active TE: {{activeTimeEntry | json}}
```
Comment (enriquezrene) [inline src/app/modules/time-entries/pages/time-entries.component.html:75]: do we need this?

File: src/app/modules/time-entries/pages/time-entries.component.html [hunk 3]
```diff
@@ -64,13 +64,15 @@ <h5 class="modal-title" id="exampleModalLongTitle">{{ entryId ? 'Edit Entry' : '
       </div>
       <div class="modal-body">
         <app-details-fields
-          [formType]="'entries'"
           [entryToEdit]="entry"
           (saveEntry)="saveEntry($event)"
-        ></app-details-fields>
+        >  Active TE: {{activeTimeEntry | json}}
```
Comment (enriquezrene) [inline src/app/modules/time-entries/pages/time-entries.component.html:69]: do we need this?

File: src/app/modules/time-entries/pages/time-entries.component.ts (hunk 1 above)
Comment (juanultimate) [reply src/app/modules/time-entries/pages/time-entries.component.ts]: we dont for sure. let me delete it.

**Pull request merged**